Dart/src/com/jetbrains/lang/dart/validation/fixes/DartServerFixIntention.java

```diff
@@ -180,9 +184,23 @@ public boolean startInWriteAction() {
   @Override
   public void invoke(@NotNull final Project project, final Editor editor, final PsiFile file) throws IncorrectOperationException {
     final SourceFileEdit fileEdit = myChange.getEdits().get(0);
+    final String filePath = fileEdit.getFile();
```

Comment:
Also SourceChange.message="Create file 'package:ProjectName/non_existing_file.dart'". For me message like "Create file 'non_existing_file.dart'" looks better.
The later is true for creating file not in lib folder: current message contains full file path like "Create file 'C:\dart_projects\DartSample2\test\a\foopart.dart'". Wouldn't "Create file 'foopart.dart'" be better?